index.html

```diff
@@ -202,12 +202,13 @@ <h2>Contact me</h2>
                     </p>
                 </div>
                 <div class="contact-form">
-                    <form class="input-field" action="https://formspree.io/f/xdobrban" method="post" >
+                    <form class="input-field" name="detailForm" action="https://formspree.io/f/xdobrban" method="post">
                         <input class="name-input" type="text" name="name" maxlength="30" required placeholder="Your name">
-                        <input class="email-input" type="email" name="email" required placeholder="Your email">
+                        <input class="email-input" type="email" name="email" id="email" required placeholder="Your email">
+                        <span class="emailerrorTxt" id="errorMsg" aria-live="polite"></span>

```

Comment:
- Let's move our error message container down under the text area or button to ensure the error message shows up close to the "submit" button to meet the project requirement.